todo_app/tests_e2e/test_e2e.py

```diff
@@ -0,0 +1,59 @@
+import os,pytest
+from threading import Thread
+from todo_app.data.trello_api import create_trello_board,delete_trello_board
+from todo_app import app
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+from dotenv import find_dotenv,load_dotenv
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    board_id = create_trello_board('TestBoard')
```

Comment:
It's worth noting that you're creating your test board _before_ loading the `.env` file - this may be a flag that the environment variables are already loaded through your test runner, since creating a board will need your key/secret for Trello. We should move this step down a couple of lines